nautobot/core/settings.py

```diff
@@ -682,7 +682,7 @@
     ),
     "SUPPORT_MESSAGE": ConstanceConfigItem(
         default="",
-        help_text="Help message to include on 4xx and 5xx error pages. Markdown is supported.\n"
+        help_text="Help message to include on 4xx and 5xx error pages. Markdown is supported, as are some HTML tags.\n"
```

Comment:
Is this accurate?
```suggestion
        help_text="Help message to include on 4xx and 5xx error pages. Markdown is supported, as are some HTML tags and attributes.\n"
```

Reply:
But then I have to wrap the help_text to another line to make black/ruff happy... :-(

Will do.